src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ScrollBarTests.cs

```diff
@@ -432,14 +432,14 @@ public unsafe void ScrollBar_Enabled_GetScrollInfo_Updates()
             };
 
             // Enable.
-            var si = new User32.SCROLLINFO
+            var si = new SCROLLINFO
             {
-                cbSize = (uint)sizeof(User32.SCROLLINFO),
-                fMask = User32.SIF.ALL
+                cbSize = (uint)sizeof(SCROLLINFO),
+                fMask = SCROLLINFO_MASK.SIF_ALL
             };
             Assert.NotEqual(IntPtr.Zero, control.Handle);
             control.Enabled = true;
-            Assert.True(User32.GetScrollInfo(control.Handle, User32.SB.CTL, ref si));
+            Assert.True(PInvoke.GetScrollInfo((HWND)control.Handle, SCROLLBAR_CONSTANTS.SB_CTL, ref si));
```

Comment:
Let's call control.HWND in this file instead of casting
```suggestion
            Assert.True(PInvoke.GetScrollInfo(control.HWND, SCROLLBAR_CONSTANTS.SB_CTL, ref si));
```